src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/custom.py

```diff
@@ -42,7 +42,7 @@ def create_vault(client, vault_name, region, resource_group_name):
     vault_sku = Sku(SkuName.standard)
     vault_properties = VaultProperties()
     vault = Vault(region, sku=vault_sku, properties=vault_properties)
-    return client.create_or_update(resource_group_name, vault_name, vault, custom_headers=_get_custom_headers())
```

Comment:
Why the removal of the custom headers?

Reply:
Earlier, we were using the custom headers to pass a modified client request ID; we were adding a suffix '-Cli' at the end of the client request ID. This was flagged in a recent discussion with Jason and thus decided to roll back the change.